net-dialup/cutecom/cutecom-0.22.0.recipe

```diff
@@ -0,0 +1,71 @@
+SUMMARY="Graphical serial terminal"
+DESCRIPTION="CuteCom is a graphical serial terminal, like minicom."
+HOMEPAGE="http://cutecom.sourceforge.net"
```

Comment:
Although some (or many) browsers and many websites don't display the trailing slashes, URIs which do not have a path part (i.e. which only have the protocol and a domain name) **need** a trailing slash in order to be valid.

Now, if a tool or webpage which displays URIs wants to hide the trailing slashes, that's perfectly OK.